fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -1126,24 +1138,26 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 			// Read and reload metadata.
 			auto mappingRange = prefixRange(persistShardMappingPrefix);
 			RangeResult rangeResult;
-			readRangeInDb(db, mappingRange, /* rowLimit= */ UINT16_MAX, /* byteLimit = */ UINT16_MAX, &rangeResult);
+			std::shared_ptr<DataShard> specialKeysShard;
```

Comment:
@yao-xiao-github , I assumed db refers to specialKeysShard and got the specialKeysShard from the shardMap. Let me know if this is not right, or if you prefer any other way.